src/js/juju-cards-v1.0.0.js

```diff
@@ -60,6 +60,12 @@ let jujuCards = () => {
     }
 
     let addLink = `${demoDomain}/?deploy-target=${getImageID(id)}`;
+    let deployTitle = 'Deploy with Juju';
```

Comment:
These two titles are pretty static so lets move them as consts to the top of the fn so they don't have to be updated in multiple places